src/app/features/issue/issue.service.ts

```diff
@@ -120,7 +120,9 @@ export class IssueService {
     }
 
     for (const issuesType of Object.keys(tasksIssueIdsByIssueType)) {
-      const updates = await (this.ISSUE_SERVICE_MAP[issuesType].refreshIssues as any)(tasksIssueIdsByIssueType[issuesType], isNotifySuccess, isNotifyNoUpdateRequired);
+      const updates = await (this.ISSUE_SERVICE_MAP[issuesType].refreshIssues as any)(tasksIssueIdsByIssueType[issuesType],
```

Comment:
This is a bit ugly I'd say. maybe we can set the max line length to a more convenient value as a quick fix? Alternatively I much prefer this:
```TypeScript
  const updates = await (this.ISSUE_SERVICE_MAP[issuesType].refreshIssues as any)( 
        tasksIssueIdsByIssueType[issuesType],
        isNotifySuccess,
        isNotifyNoUpdateRequired
  );
```

Reply:
Should be ok now